react/src/reactTextMask.js

```diff
@@ -52,12 +37,12 @@ export const MaskedInput = React.createClass({
     return (
       <input
         {...props}
-        onInput={this.onChange}
+        onInput={event => this.onChange(event)}
```

Comment:
Yes, createClass autobinds methods on the components, classes do not. So you have to use an arrow function to keep the value of `this`